<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageManagerLog</class>
 <widget class="QWidget" name="MessageManagerLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="statusTip">
   <string>Full messages log|This tab allows you to see all the sent and received messages. Useful to see exactly what's happening and to understand IanniX inputs/outputs.\n Be careful, this log is *very* CPU intensive and the score accuracy may decrease if this tab is displayed during playback.</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#logSend, QWidget#logReceive {
	font: 						10px &quot;Lucida Console&quot;, &quot;Monaco&quot;, &quot;Monospace&quot;;
	background-color: 	rgb(242, 241, 237);
	color:						black;
}
QPushButton  {
	min-width:  		14px;
	min-height: 		14px;
	max-width:  	14px;
	max-height: 	14px;
	width:  				14px;
	height: 			14px;
	padding:			0px;
	border-radius: 7px;
}
QPushButton#logSendCopy, QPushButton#logReceiveCopy {
	min-width:  		40px;
	max-width:  	40px;
	width:  				40px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget23">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>SENT MESSAGES (only when this tab is visible)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="logSendCopy">
           <property name="toolTip">
            <string>Copies sent messages log to clipboard</string>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="logSendClear">
           <property name="toolTip">
            <string>Clears sent messages log</string>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="logSend">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopWidth">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>RECEIVED MESSAGES (only when this tab is visible)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="logReceiveCopy">
           <property name="toolTip">
            <string>Copies received messages log to clipboard</string>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="logReceiveClear">
           <property name="toolTip">
            <string>Clears received messages log</string>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="logReceive">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="tabStopWidth">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>logSendClear</sender>
   <signal>released()</signal>
   <receiver>logSend</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logReceiveClear</sender>
   <signal>released()</signal>
   <receiver>logReceive</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logSendCopy</sender>
   <signal>released()</signal>
   <receiver>MessageManagerLog</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logReceiveCopy</sender>
   <signal>released()</signal>
   <receiver>MessageManagerLog</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action()</slot>
 </slots>
</ui>
